### DISCUSSION TITLE
Availability - 1

### DISCUSSION THREAD

**Discussion by changjurhee** [changjurhee/AONE#13] [Polls]
## Quality Attributes

Type | Description | QA Scenario (Stimulus / Source of stiulus / Artifact / Environment / Response / Response measure) | Tatics
 -- | -- | -- | --
Availability | Provide reliable signaling for both inbound and outbound calls including dropped call detection. | When a user makes or terminates a direct call using the system in the runtime environment, the frequency of accurately responding to the call / end call attempt must be 99.9% or higher.<br>- Make or Terminate call<br>- User<br>- Call service<br>- Runtime<br>- Respond normally<br>- More than 99% normal response rate |

**Comment by codekistone** [comment]
Manage work requests - Even if the system is busy with other tasks, when it receives incoming call and call hanging up event, it needs to prioritize processing the request.
